src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Cohost/Shared/CohostFindAllReferencesEndpointTest.cs

```diff
@@ -4,6 +4,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Razor;
+using Microsoft.AspNetCore.Razor.Language;
```

Comment:
Is this needed? I don't see any new type or extension references from this namespace in the code below.